docs/src/howto-guides/025_config_management_plugin.md

```diff
@@ -0,0 +1,129 @@
+```
+
+### Using the `c8y_configuration_plugin` Debian package for Debian OS distribution
+
+For Debian OS distribution, we provide the `c8y_configuration_plugin_<version>_<arch>.deb` package as a release asset [here](https://github.com/thin-edge/thin-edge.io/releases).
+
+In case that you didn't use the `get-thin-edge_io.sh` script, you can download the `c8y_configuration_plugin_<version>_<arch>.deb`  package on our [Releases](https://github.com/thin-edge/thin-edge.io/releases) and install it.
+
+```shell
+sudo apt install ./path/to/package/c8y_configuration_plugin_<version>_<arch>.deb
+```
+
+### Extracting from debian package for non-Debian OS distribution
+
+Get the `c8y_configuration_plugin_<version>_<arch>.deb` from our [Releases](https://github.com/thin-edge/thin-edge.io/releases).
+Then, run this command in the directory where the package is stored.
+
+```shell
+ar -x ./c8y_configuration_plugin_<version>_<arch>.deb | tar -xf ./data.tar.xz
+```
+
+The binary is extracted in `<current directory>/usr/bin`.
+For more details, refer to our guide [Extracting from debian package](./015_installation_without_deb_support.md#extracting-binaries-from-deb-packages).
+
+### Building from sources
+
+Follow our guide [Buiding thin-edge.io](./../BUILDING.md) and [Building from source](./015_installation_without_deb_support.md#if-building-from-source).
+
+```shell
+cargo build --release -p c8y_configuration_plugin
+```
+
+A `systemd` unit file for `c8y_configuration_plugin` can be found in the repository at `configuration/init/systemd/c8y-configuration-plugin.service`
+and should be installed on the target in: `/lib/systemd/system/c8y-configuration-plugin.service`.
+
+```shell
+sudo cp <repository_root>/configuration/init/systemd/c8y-configuration-plugin.service /lib/systemd/system/c8y-configuration-plugin.service
+```
+
+## Get started
+
+Before starting anything, make sure [your device is connected to Cumulocity](./../tutorials/connect-c8y.md).
+
+**Step 0**
+Only if you didn't install `c8y_configuration_plugin` by the debian package,
+you need one additional step to initialize the plugin. Run this command.
+
+```shell
+sudo c8y_configuration_plugin --init
+```
+
+**Step 1**
+Open the file `/etc/tedge/c8y/c8y-configuration-plugin.toml` and edit the file.
+
+```toml
+files = [
+    { path = '/etc/tedge/tedge.toml' },
+    { path = '/etc/tedge/mosquitto-conf/c8y-bridge.conf', type = 'c8y-bridge.conf' },
+    { path = '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf', type = 'tedge-mosquitto.conf' },
+    { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto.conf' },
+    { path = '/etc/tedge/c8y/example.txt', type = 'example', user = 'tedge', group = 'tedge', mode = 0o444 }
+]
+```
+
+Add the files you want to configure from the cloud using the [format described in the reference guide](./../references/c8y-configuration-management.md#configuration).
```

Comment:
```suggestion
* `path` is the full path to the configuration file.
* `type` is a unique alias for each file entry which will be used to represent that file in Cumulocity UI.
* `user`, `group` and `mode` are UNIX file permission settings to be used to create a configuration file. If not provided, the files will be created with `root` user. If the file exists already, its ownership will be retained.

For more details on this configuration file format, refer to the [reference guide](./../references/c8y-configuration-management.md#configuration).
```

Reply:
Indeed, easier for the user to have the key points inline and a link for the details,